### ISSUE TITLE
red-connector-ssh does not use port

### ISSUE THREAD

**Issue by Domprobst** [curious-containers/curious-containers#122]
**Description**

When connecting to a remote host using a private key instead of a password, the specified port is not utilized. Consequently, it becomes impossible to connect to any port other than the default one.

```
try:
    client.connect(host, username=username, pkey=pkey)
except socket.gaierror:
```

**Comment by Domprobst** [comment]
merged with pull request red-connector-ssh #3

**Comment by Domprobst** [comment]
released with https://github.com/curious-containers/red-connector-ssh/releases/tag/1.3
